envoy/ssl/connection.h

```diff
@@ -103,6 +103,18 @@ class ConnectionInfo {
    **/
   virtual absl::Span<const std::string> dnsSansLocalCertificate() const PURE;
 
+  /**
+   * @return absl::Span<const std::string> the IP entries in the SAN field of the peer certificate.
+   *         Returns {} if there is no peer certificate, or no SAN field, or no IPs.
+   **/
+  virtual absl::Span<const std::string> ipSansPeerCertificate() const PURE;
```

Comment:
Pass-by comment: is it worth presenting them as IPs not strings? Do you need to do some wildcard matching?

Reply:
I'd say keep them as strings, unless there's a good reason to convert them. They start out as strings in the certificate, and right now we don't have to parse them at all.

On the other hand, there may be subtle formatting differences that could be avoided by converting them to IP addresses internally.